### DISCUSSION TITLE
Status badge stuck on passing

### DISCUSSION THREAD

**Discussion by ay0o** [community/community#25998] [Actions]
<p>Just wrote my first GitHub Action, everything seems correct except that the status badge is stuck on “passing” even though the workflow successfully finished a while ago. Any idea why?</p>
<p><a href="https://github.com/elemenophy/pricetopus/workflows/.github/workflows/ci.yaml/badge.svg" class="onebox" target="_blank" rel="nofollow noopener">https://github.com/elemenophy/pricetopus/workflows/.github/workflows/ci.yaml/badge.svg</a><br>
</p><aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/elemenophy/pricetopus/actions" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/23629729/181094464-36c40a91-9b6b-49a0-a2d9-df3a459becb2.jpeg" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/elemenophy/pricetopus/actions" target="_blank" rel="nofollow noopener">elemenophy/pricetopus</a></h3>

<p>Contribute to elemenophy/pricetopus development by creating an account on GitHub.</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by airtower-luna** [comment, marked as answer]
<p>“Passing” in this context generally means that the check executed successfully, as in “the test passed”, so I’d consider that expected behavior.</p>

**Comment by ay0o** [comment]
<p>ah, I was expecting a “done”, “success”. Ok, then :slight_smile:</p>
